update.ts

```diff
@@ -0,0 +1,61 @@
+import * as semver from "https://deno.land/x/semver@v1.0.0/mod.ts";
+import {
+  writeJson,
+} from "https://deno.land/std@0.61.0/fs/mod.ts";
+
+import { VERSION } from "./version.ts";
+
+const nestAPI = "https://x.nest.land/api/package/std";
+
+const egg: any = await fetch(nestAPI)
+  .then((response) => response.json())
+  .then((data) => {
+    return semver.clean(data.latestVersion.replace("std@", ""));
+  });
+
+console.log("nest.land: ", egg);
+console.log("deno.land: ", VERSION);
+
+const diff: number = semver.compare(egg, VERSION);
+
+const config: object = {
+  "name": "std",
+  "description": "A decentralized mirror of Deno's Standard Modules",
+  "version": VERSION,
+  "entry": "./version.ts",
+  "stable": false,
+  "unlisted": true,
+  "fmt": false,
+  "repository": "https://github.com/denoland/deno/tree/master/std",
```

Comment:
```suggestion
  "repository": "https://github.com/nestdotland/std",
```

I think we should keep the nest.land std repository here so people know how we're doing this.

Reply:
ok 👍